keyboards/whitefox/whitefox.h

```diff
@@ -70,4 +70,36 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     { K80, K81, K82, K83, K84,   K85,   K86, KC_NO }  \
 }
 
+/* ISO WhiteFox
+ * ,---------------------------------------------------------------.
+ * |Esc|  1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =| Backsp|Ins|
+ * |---------------------------------------------------------------|
+ * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|Enter|Del|
+ * |------------------------------------------------------`    |---|
+ * |CapsLo|  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|  \|    |PgU|
+ * |---------------------------------------------------------------|
+ * |Shif|   |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /|Shift |Up |PgD|
+ * |---------------------------------------------------------------|
+ * |Ctrl|Gui |Alt |         Space    |Fn0 |Alt |Gui |  |Lef|Dow|Rig|
+ * `---------------------------------------------------------------'
+ */
+
+#define KEYMAP_ISO( \
```

Comment:
Please change this to `LAYOUT_iso`

Reply:
And, if you could, change `KEYMAP` to `LAYOUT_all` and `KEYMAP_TRUEFOX` to `LAYOUT_truefox`?